derp/derp_test.go

```diff
@@ -398,14 +400,14 @@ func TestSendFreeze(t *testing.T) {
 	}
 	drain := func(t *testing.T, name string) bool {
 		t.Helper()
-		timer := time.NewTimer(1 * time.Second)
+		timer, timerChannel := clock.NewTimer(1 * time.Second)
```

Comment:
This one is preventing the test from running forever, probably need that to still be a normal clock